### ISSUE TITLE
Outlook 2016: When NVDA is set to launch after Windows is started, read/unread status of messages is not indicated

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#6874]
NVDA does not read the unread status of messages in Outlook first launch after Win boot. This problem has existed for awhile. I cannot say when it started or whether it is limited to Windows 10. See steps and log for confirmation. Essentially, when Outlook is first launched after Windows is started, Read/unread status of messages is not reliable. Every time, NVDA indicates messages to be read. This is solved if NVDA is restarted.

* Expected result: NVDA should read the read/unread status regardless of when NVDA is started.
* Actual result: NVDA reads all messages as if they are read.

STR:
1. Set NVDA to start upon login.
2. Reboot Windows
3. Launch Outlook
4. Try to read the read/unread status of all messages.

IO - speech.speak (10:55:03):
Speaking [LangChangeCommand ('en'), u'level 1  collapsed From Annemarie Cooke;  Dan Rossi;  Mike Pietruk;  Dan Tevelde;  Catherine Getchell, Subject [VICUG-L] Know of a Really Inaccessible Website?, Received Sat 2/11/2017 10:47 AM, Size 35 KB,  collapsed  row 2  column 1']
IO - inputCore.InputManager.executeGesture (10:55:03):
Input: kb(laptop):downArrow
IO - speech.speak (10:55:03):
Speaking [LangChangeCommand ('en'), u'From delivery.com, Subject Love a deal? Get 10% off this weekend., Received Sat 2/11/2017 10:40 AM, Size 17 KB,  row 3  column 1  level 1']
IO - inputCore.InputManager.executeGesture (10:55:04):
Input: kb(laptop):downArrow
IO - speech.speak (10:55:04):
Speaking [LangChangeCommand ('en'), u"From ProFlowers, Subject Hot Valentine's Day Deals: 57% OFF! Don't miss them!, Received Sat 2/11/2017 10:16 AM, Size 47 KB,  row 4  column 1  level 1"]
IO - inputCore.InputManager.executeGesture (10:55:06):
Input: kb(laptop):downArrow
IO - speech.speak (10:55:06):
Speaking [LangChangeCommand ('en'), u'From DNC War Room, Subject TRUMP POLL UPDATE FOR PRATIK:, Received Sat 2/11/2017 9:34 AM, Size 20 KB,  row 5  column 1  level 1']
IO - inputCore.InputManager.executeGesture (10:55:07):
Input: kb(laptop):downArrow
IO - speech.speak (10:55:07):
Speaking [LangChangeCommand ('en'), u'From Facebook, Subject Maia Scott added a new photo, Received Sat 2/11/2017 8:38 AM, Size 20 KB,  row 6  column 1  level 1']
IO - inputCore.InputManager.executeGesture (10:55:09):
Input: kb(laptop):downArrow
IO - speech.speak (10:55:09):
Speaking [LangChangeCommand ('en'), u'collapsed From Damian A. Sian;  Knox, Marg;  Landeros, Estela;  Jess Thompson;  KRISTA L. GREEAR;  Thompson, Rachel, Subject [ITACCESS] Word to PDF conversion?, Received Sat 2/11/2017 8:35 AM, Size 145 KB,  collapsed  row 7  column 1  level 1']
IO - inputCore.InputManager.executeGesture (10:55:12):
Input: kb(laptop):alt+n
IO - speech.speak (10:55:12):
Speaking [LangChangeCommand ('en'), u'collapsed From Damian A. Sian;  Knox, Marg;  Landeros, Estela;  Jess Thompson;  KRISTA L. GREEAR;  Thompson, Rachel, Subject [ITACCESS] Word to PDF conversion?, Received Sat 2/11/2017 8:35 AM, Size 145 KB,  collapsed  row 7  column 1  level 1']

**Comment by ehollig** [comment]
I am not able to reproduce this using Outlook 2016 and the latest Master of NVDA. @PratikP1, is this still happening for you?

**Comment by PratikP1** [comment]
I am no longer having this issue. It is resolved.

**Comment by ehollig** [comment]
Closing because of https://github.com/nvaccess/nvda/issues/6874#issuecomment-323025282
